src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -527,7 +531,12 @@ private void errorHandler(RoutingContext routingContext) {
 
     private void metricsHandler(RoutingContext routingContext) {
         PrometheusMeterRegistry prometheusMeterRegistry = (PrometheusMeterRegistry) meterRegistry;
-        routingContext.response().setStatusCode(HttpResponseStatus.OK.code()).end(prometheusMeterRegistry.scrape());
+
+        StringBuilder sb = new StringBuilder();
+        sb.append(jmxCollectorRegistry.scrape());
+        sb.append(prometheusMeterRegistry.scrape());
+
+        routingContext.response().setStatusCode(HttpResponseStatus.OK.code()).end(sb.toString());
```

Comment:
I wonder if this can be somehow put into one place instead of having it twice - here and in the EmbeddedHttpServer.

Reply:
Agree I did a little bit of refactoring adding a common `MetricsReporter` class to be used by both.